test/fixtures/folder/client.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
Do the fixtures need to be javascript files? Might be nice to avoid having JS so that the NPM script doesn't need to be changed and a single test can be run with `npm run test -- [some test file]`

Reply:
For single files one normally just run `node path/to/file.js` with tap. Tap differ here from tools like jest and mocha.

Reply:
I tried that but the output is a bit hard to read compared to using the tap command.

Reply:
I've fixed this in a different way. `tap --test-regex=.test.js --test-ignore=node_modules`
Now you can do `npm test` or `npm test -- test/classes/eik-config.test.js` as you like.